careers/_template-students-and-graduates.html

```diff
@@ -0,0 +1,9 @@
+<h2 class="h4">Students & Recent Graduates</h2>
+<p class="short-desc">
+    Find opportunities in public service for a new generation of leaders.
+</p>
+<p class="short-desc">
+    <a href="/careers/students-and-graduates/" class="jump-link jump-link__right">
+        Learn about students & graduates opportunities
```

Comment:
@sarahsimpson09 let me know if I should go with @Scotchester edit for the link. I'm merging this PR, but a copy edit here can be caught in further PRs since we have additional careers pages in the pipe.